Symantec dlp modeling (#32691)
demisto/content: Packs/SymantecDLP/README.md

- View incident details in the default layout, such as Incident Type, Severity, Detection Server Name, Group Name, Policy Violation, etc.
- Protect data from being exposed or stolen in real-time.
- Monitor user activity to ensure that they do not send sensitive or critical information outside the corporate network. 

<~XSIAM>
# Symantec Data Loss Prevention
This pack includes Cortex XSIAM content.

## Configuration on Server Side
**Note:**
<br>
If you configure both **Syslog Server Alerts** and **Syslog Response Rule**, make sure to configure the: 
- port
- protocol 
- host 

of the syslog server the same.

> If you want to configure them differently, make sure to configure 2 different syslog data source on XSIAM as well, by repeating the **broker vm configuration** steps for each one.

Reviewer: ```suggestion
If you want to configure them differently, make sure to configure 2 different syslog data sources on XSIAM as well, by repeating the [#Broker VM](broker-vm) procedure for each one.

```

<br>

### Syslog Response Rule

#### Configure a Log to a Syslog Server action
In order to send logs via syslog, you will need to create a response rule with a "Log to a Syslog Server" action.
1. In the Enforce Console, navigate to **Manage** &rarr; **Policies** &rarr; **Response Rules**.

> 2. Click on **Add Response Rule**.

Reviewer: ```suggestion
2. Click **Add Response Rule**.

```

3. Leave the type of response rule as **Automated Response** and click **Next**.

> 4. Enter the below:

Reviewer: ```suggestion
4. Enter the following:

```

    - **Rule Name** - "Cortex XSIAM Syslog" 

>     - **Description** - "This response rule sends is configured to send logs via syslog, to the Cortex XSIAM Broker VM.".

Reviewer: ```suggestion
    - **Description** - "This response rule is configured to send logs via syslog to the Cortex XSIAM Broker VM.".

```

5. Select **Log to a Syslog Server** from the Actions dropdown list and click **Add Action**.

> 6. Enter the Host name or IP Address of the syslog server under **Host**.

Reviewer: ```suggestion
6. Enter the Host name or IP address of the syslog server under **Host**.

```

> 7. Edit the Port for the syslog server, if necessary.

Reviewer: ```suggestion
7. Edit the port for the syslog server, if necessary.

```

8. Select a communication protocol.
   You can select UDP or TCP.

>    If you select TCP, you can secure communications to the syslog server by selecting Enable TLS Client Authentication.

Reviewer: ```suggestion
   If you select TCP, you can secure communications to the syslog server by selecting **Enable TLS Client Authentication**.

```

> 9. Copy and paste the below configuration under **Message**:

Reviewer: ```suggestion
9. Copy and paste the following configuration under **Message**:

```

    ```bash
   "APPLICATION_NAME":"$APPLICATION_NAME$", "APPLICATION_USER":"$APPLICATION_USER$", "ATTACHMENT_FILENAME":"$ATTACHMENT_FILENAME$", "BLOCKED":"$BLOCKED$", "DATAOWNER_NAME":"$DATAOWNER_NAME$", "DATAOWNER_EMAIL":"$DATAOWNER_EMAIL$", "DESTINATION_IP":"$DESTINATION_IP$", "ENDPOINT_DEVICE_ID":"$ENDPOINT_DEVICE_ID$", "ENPOINT_LOCATION":"$ENDPOINT_LOCATION$", "ENDPOINT_MACHINE":"$ENDPOINT_MACHINE$", "ENDPOINT_USERNAME":"$ENDPOINT_USERNAME$", "PATH":"$PATH$", "FILE_NAME":"$FILE_NAME$", "PARENT_PATH":"$PARENT_PATH$", "INCIDENT_ID":"$INCIDENT_ID$", "MACHINE_IP":"$MACHINE_IP$", "MATCH_COUNT":"$MATCH_COUNT$", "OCCURRED_ON":"$OCCURRED_ON$", "POLICY":"$POLICY$", "RULES":"$RULES$", "PROTOCOL":"$PROTOCOL$", "REPORTED_ON":"$REPORTED_ON$", "SCAN":"$SCAN$", "SENDER":"$SENDER$", "MONITOR_NAME":"$MONITOR_NAME$", "SEVERITY":"$SEVERITY$", "STATUS":"$STATUS$", "SUBJECT":"$SUBJECT$", "TARGET":"$TARGET$", "USER_JUSTIFICATION":"$USER_JUSTIFICATION$", "URL":"$URL$", "RECIPIENTS":"$RECIPIENTS$", "INCIDENT_SNAPSHOT":"$INCIDENT_SNAPSHOT$"
   ```

>    This will format the log's message in a json format.

Reviewer: ```suggestion
   This will format the log's message in JSON format.

```

> 10. Select the **Informational** Level from the **Level** dropdown list.

Reviewer: ```suggestion
10. Select **Informational** from the **Level** dropdown list.

```

11. Click **Save**.
    
#### Assign a response rule to a policy
After creating the response rule with the "Log to a Syslog Server" action, you will have to assign it to a policy in order for it to apply.
1. In the Enforce Console, navigate to **Manage** &rarr; **Policies** &rarr; **Policy List**.
2. Select the policy you want to collect logs from using syslog.

> 3. Go to the **Response** tab, and select the **Cortex XSIAM Syslog** rule you have created in the previous section.

Reviewer: ```suggestion
3. Go to the **Response** tab, and select the **Cortex XSIAM Syslog** rule you created in the previous section.

```

4. Click **Add Response Rule** to add the response rule to the policy.
5. Click **Save**
6. Repeat steps 2-5 for any additional policies you want to collect logs from and send to Cortex XSIAM via syslog.

<br>

See [Configuring the Log to a Syslog Server action](https://techdocs.broadcom.com/us/en/symantec-security-software/information-security/data-loss-prevention/15-8/about-response-rules-v40462357-d339e11/configuring-the-log-to-a-syslog-server-action-v27591174-d339e4495.html) for additional details.

###  Syslog Server Alerts

> To enable sending server events to the syslog server, follow the below steps:

Reviewer: ```suggestion
To enable sending server events to the syslog server, complete the following steps:

```

1. Go to the \Program Files\Symantec\DataLossPrevention\EnforceServer\15.8.00000\Protect\config directory on Windows or the /opt/Symantec/DataLossPrevention/EnforceServer/15.8.00000/Protect/config directory on Linux.
2. Open the Manager.properties file.

> 3. Uncomment the #systemevent.syslog.protocol = line by removing the # symbol from the beginning of the line, and enter [ udp | tcp | tls ] to secure communications sent from the Enforce Server to the syslog server.

Reviewer: ```suggestion
3. Uncomment the **#systemevent.syslog.protocol =** line by removing the # symbol from the beginning of the line, and enter **[ udp | tcp | tls ]** to secure communications sent from the Enforce Server to the syslog server.

```

> 4. Uncomment the #systemevent.syslog.host= line by removing the # symbol from the beginning of the line, and enter the hostname or IP address of the syslog server.
> 5. Uncomment the #systemevent.syslog.port= line by removing the # symbol from the beginning of the line. Enter the port number that should accept connections from the Enforce Server. The default is 514.
> 6. Uncomment the #systemevent.syslog.format= [{0}] {1} - {2} line by removing the # symbol from the beginning of the line.

Reviewer: ```suggestion
4. Uncomment the **#systemevent.syslog.host= line** by removing the # symbol from the beginning of the line, and enter the hostname or IP address of the syslog server.
5. Uncomment the **#systemevent.syslog.port= line** by removing the # symbol from the beginning of the line. Enter the port number that should accept connections from the Enforce Server. The default is 514.
6. Uncomment the **#systemevent.syslog.format= [{0}] {1} - {2}** line by removing the # symbol from the beginning of the line.

```
